src/App.js

```diff
@@ -1,27 +1,11 @@
-import logo from './logo.svg';
+// import logo from './logo.svg';
 import './App.css';
+import MyComponent from './components/Calculator';
```

Comment:
- [ ] It looks like you're using `MyComponent` to declare the component. As you can see in your project, this works, but it could be confused if we have an application with 100 components, please, in place to use `MyComponent` as the component name, use `Calculator` to have better practices on your react application. 🦊